lyse/__init__.py

```diff
@@ -703,12 +705,21 @@ def globals_groups(self):
             except KeyError:
                 return []   
                 
-    def globals_diff(self, other_run, group=None):
-        return globals_diff(self, other_run, group)            
+    def globals_diff(self, other_shot, group=None):
+        return globals_diff(self, other_shot, group)            
     
-        
-class Sequence(Run):
-    def __init__(self, h5_path, run_paths, no_write=False):
+
+class Run(Shot):
+    def __init__(self, *args, **kwargs):
+        warnings.warn("The 'Run' class has been renamed to 'Shot' (but is otherwise compatible). 'Run' will be removed in lyse v4.0. Please update your analysis code to use the 'Shot` class", DeprecationWarning)
```

Comment:
This line is pretty long; maybe use `dedent()`. Also it looks like the last occurrence of the word 'Shot' has one single quote and one backtick instead of two single quotes, and the second sentence is missing a period. Hope that's not too obnoxious and nitpicky.

Reply:
Thanks. This is perfect (and now fixed in 016768d)